pages/airdrop/claim.vue

```diff
@@ -158,19 +155,14 @@ export default class AirdropClaimPage extends Vue {
       try {
         const res: any = await this.$axios
           .get(`${AIRDROP_CLAIM}${this.currentAddress}`)
-          .catch((err) => {
-            if (err.response)
-              this.errorMessage = this.$t(
-                'AirDrop.errorMessage.noAddress',
-              ) as string
-          })
-        this.totalAirdrop = Math.round(
-          (res.data.reward.unclaimedAmount + res.data.reward.claimedAmount) *
-            Denom.Nanolike,
-        )
-        this.totalClaimedAmount = Math.round(
-          res.data.reward.claimedAmount * Denom.Nanolike,
-        )
+
+        this.totalAirdrop = new BigNumber(
+          res.data.reward.unclaimedAmount + res.data.reward.claimedAmount,
+        ).shiftedBy(-9).integerValue() 
+        this.totalClaimedAmount = new BigNumber(
+          res.data.reward.claimedAmount,
+        ).shiftedBy(-9).integerValue() 
```

Comment:
Are these expected to be 100% integer? Should we use round_down mode?

Reply:
Think we keep 100% integer is better for UI.

Reply:
Should use `BigNumber.ROUND_DOWN` mode I guess